### PR TITLE
enable Dependabot v2

### PR DESCRIPTION
**Pull request by sullis**
https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/
﻿

### CODE REVIEW

**Review by mstachniuk — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "github-actions"
```
Comment (mstachniuk) [inline .github/dependabot.yml:7]: Shipkit doesn't use github actions

**Review by mstachniuk — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (sullis) [reply .github/dependabot.yml:7]: > Shipkit doesn't use github actions

is Shipkit going to migrate away from TravisCI?  

The Mockito-Core project appears to be moving to GitHub Actions:
 
https://github.com/mockito/mockito/tree/release/3.x/.github/workflows

**Comment by mockitoguy** [comment]
Hey! We're putting the project in maintenance mode soon. Dependabot opens PRs that won't be merged. Let's skip that PR for now.

@sullis, do you want to add dependabot to https://github.com/shipkit/shipkit-auto-version ?

**Comment by sullis** [comment]
> Hey! We're putting the project in maintenance mode soon. Dependabot opens PRs that won't be merged. Let's skip that PR for now.
> 
> @sullis, do you want to add dependabot to https://github.com/shipkit/shipkit-auto-version ?

new PR:  https://github.com/shipkit/shipkit-auto-version/pull/45

**Pull request closed without merging**